transform/mattermost-analytics/models/reports/product/active_user_base/_active_user_base___models.yml

```diff
@@ -34,8 +34,35 @@ models:
         description: Last known IP reported by the server either via telemetry or diagnostics.
       - name: last_known_ip_country
         description: Country for the last known IP address.
-      - name: last_known_server_ip_date
-        description: Date that the last known server IP was recorded.
+      - name: installation_type
+        description: |
+          The type of installation for this server. Values can be `deb_package`, `docker`, `gitlab_omnibus`, 
```

Comment:
might include 'gitlab' as well.

Reply:
It's actually not in the values in the table.

Reply:
I see it here - https://docs.google.com/spreadsheets/d/1kzfASpc-pyfwavkEIMP-9xGzgkZdxo2XBlJ8L4JYsLE/edit#gid=2131338777 for some reason - maybe old repo set value?

Reply:
- `gitlab_omnibus` is the value send by telemetry. It seems to be set here: https://github.com/gitlabhq/omnibus-gitlab/blob/master/files/gitlab-cookbooks/mattermost/libraries/mattermost_helper.rb#L49 by gitlab's omnibus package.
- `gitlab` means that (a) installation type is null (missing telemetry, bug in server fact etc) and (b) gitlab oauth is enabled on the server ([source](https://github.com/mattermost/mattermost-data-warehouse/blob/master/transform/snowflake-dbt/models/mattermost/hourly/server_fact.sql#L377)).